src/topology/algebra/order/basic.lean

```diff
@@ -3,11 +3,9 @@ Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro, Yury Kudryashov
 -/
-import data.set.intervals.pi
 import data.set.pointwise.interval
-import order.filter.interval
 import topology.algebra.group.basic
-import topology.algebra.order.left_right
+import topology.order.basic
 
 /-!
 # Theory of topology on ordered spaces
```

Comment:
Can you make sure that this docstring doesn't duplicate `topology/order/basic.lean`? It looks like right now they're the same as each other.

It looks to me like this docstring is basically all about the non-algebraic file.